zebedee/client_test.go

```diff
@@ -748,11 +740,7 @@ func TestUnitClient(t *testing.T) {
 					So(r.RelatedDatasets[0].Summary, ShouldEqual, "This is the page summary")
 					So(len(r.RelatedAPIDatasets), ShouldEqual, 2)
 					So(r.RelatedAPIDatasets[0].URI, ShouldEqual, "cantabularPageDescription")
-					So(r.RelatedAPIDatasets[0].Title, ShouldEqual, "Census data for London - population by age: 2021")
-					So(r.RelatedAPIDatasets[0].Summary, ShouldEqual, "Population by age in the boroughs of London")
 					So(r.RelatedAPIDatasets[1].URI, ShouldEqual, "cmdPageDescription")
-					So(r.RelatedAPIDatasets[1].Title, ShouldEqual, "A CMD dataset Title: singular")
```

Comment:
We should also be able to set the title for the CMD dataset in line 108 E.g. "relatedAPIDatasets":[{"title": "cantabularDatasetTitle","uri":"cantabularPageDescription"},{"title": "cmdDatasetTitle","uri":"cmdPageDescription"}],